templates/email/default/_email_comment_list.html

```diff
@@ -5,7 +5,7 @@
         <img style="[% list_item_photo_style %]" src="[% inline_image(update.get_first_image_fp) %]" alt="">
       </a>
     [%~ END %]
-      [% update.item_text | html_para | replace('<p>', '<p style="' _ list_item_p_style _ '">') %]
+      [% email_html_sanitize(update) | html_para | replace('<p>', '<p style="' _ list_item_p_style _ '">') %]
```

Comment:
Guess one issue is things like links won't have the right link style, I assume (like we fix the `<p>` in this line) - could fix that one too at least, I guess, not sure if anything else would matter. Probably minor.

Reply:
Doesn't look like we apply a `class` or `style` to links, just a top-level directive for `a` in the document CSS. I suppose if the user has put their own `style` attribute it might clash with the top-level rule... Probably OK, like you say, but I guess we could strip the `style` only in email HTML?